Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:14.04
+
+MAINTAINER Doody Parizada doody.parizada @ gmail.com
+
+# install apt dependencies
+RUN apt-get update -y && \
+    apt-get install -y build-essential git python python-dev python-setuptools python-pip && \
+    apt-get build-dep -y python-imaging && \
+    apt-get build-dep -y python-lxml && \
+    pip install --upgrade pip
+
+
+RUN mkdir -p /opt/knesset && mkdir -p /opt/database
+
+# get test database file
+WORKDIR /opt/database
```

Comment:
the comment is misleading. `WORKDIR` just changes the working directory and doesn't get anything